C/plugins/north/OMF/include/linkedlookup.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _LINKEDLOOKUP_H
+#define _LINKEDLOOKUP_H
+typedef enum {
+	OMFBT_UNKNOWN, OMFBT_DOUBLE64, OMFBT_DOUBLE32, OMFBT_INTEGER16,
+	OMFBT_INTEGER32, OMFBT_INTEGER64, OMFBT_UINTEGER16, OMFBT_UINTEGER32,
+	OMFBT_UINTEGER64, OMFBT_STRING, OMFBT_FLEDGEASSET
+} OMFBaseType;
+
+/**
+ * Linked Asset Information class
+ *
+ * This is the data stored for each asset and asset datapoint pair that
+ * is being sent to PI using the linked container mechanism. We use the class
+ * so we can comnbine all the information we need in a single lookup table,
+ * this not only saves space but allows to to build and retian the table
```

Comment:
Typo: retian -> retain

Reply:
There has been an edit but there is still a typo. _Retian_ became _reitan_. Also just noticed "..allows to _**to**_ build and..." in the same sentence.